stdlib/public/Darwin/Foundation/Data.swift

```diff
@@ -17,7 +17,7 @@ import Darwin
 #elseif os(Linux)
 import Glibc
 
-@inlinable
+@inlinable // This is @inlinable as trivially computable.
```

Comment:
I audited every usage of `@inlinable` and `@usableFromInline` and gave each one a justification for keeping it or changing it. I tried to be consistent with explanations.